src/utils.ts

```diff
@@ -38,15 +38,10 @@ export async function wrapNative(amount: BN, walletPublicKey: PublicKey, connect
   return tx;
 }
 
-export async function findAssociatedTokenAddress(
-  walletAddress: PublicKey,
-  tokenMintAddress: PublicKey
-): Promise<PublicKey> {
-  return (
-    await PublicKey.findProgramAddress(
-      [walletAddress.toBuffer(), TOKEN_PROGRAM_ID.toBuffer(), tokenMintAddress.toBuffer()],
-      ASSOCIATED_TOKEN_PROGRAM_ID
-    )
+export function findAssociatedTokenAddress(walletAddress: PublicKey, tokenMintAddress: PublicKey): PublicKey {
```

Comment:
Can use spl-token method? ex: https://solana-labs.github.io/solana-program-library/token/js/modules.html#getAssociatedTokenAddress

Reply:
by affect you mean changing this method to a sync function?

Reply:
Oh, I meant this util seems to be redundant and can be removed since we can use `[getAssociatedTokenAddress](https://solana-labs.github.io/solana-program-library/token/js/modules.html#getAssociatedTokenAddress)` from `spl-token`

Reply:
Cuz `getAssociatedTokenAddress` from `spl-token` is an async function, that's why I choose to change the `findAssociatedTokenAddress` to sync which is better imo